src/Microsoft.NetCore.Analyzers/UnitTests/Security/ReviewCodeForDllInjectionVulnerabilitiesTests.cs

```diff
@@ -0,0 +1,125 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.NetCore.Analyzers.Security.UnitTests
+{
+    using System;
+    using Microsoft.CodeAnalysis;
+    using Microsoft.CodeAnalysis.Diagnostics;
+    using Microsoft.CodeAnalysis.Testing;
+    using Microsoft.NetCore.Analyzers.Security;
+    using Test.Utilities;
+    using Xunit;
```

Comment:
Nit: Not a big deal but we generally put the usings outside namespace in the repo - I am not going to be pedantic about it if you prefer this style.

Reply:
I personally prefer usings outside of namespaces too.  Guess I picked up the habit from a previous project.